runtime/autoload/man.vim

```diff
@@ -8,6 +8,8 @@ try
   if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~# '^5'
     let s:man_find_arg = '-l'
   endif
+  call system('env MANPAGER=cat man -l ' . system('man -w man') . ' > /dev/null')
```

Comment:
`s:system` fails with "Unknown function: <SNR>71_system" probably because it's not in scope yet. But I certainly gave up on understanding how VimL scoping actually works.